R/read_extract_prescribing.R

```diff
@@ -6,7 +6,7 @@
 read_extract_prescribing <- function(year, file_path = get_it_prescribing_path(year)) {
   pis_file <- read_file(file_path,
     col_type = readr::cols_only(
-      "Pat UPI [C]" = readr::col_character(),
+      "anon_chi" = readr::col_character(),
```

Comment:
```suggestion
      "Pat UPI [C]" = readr::col_character(),
```
Keep IT extracts read in as chi for now